### DISCUSSION TITLE
How can I change the Ant Media Server Settings?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5558] [FAQs]
I was looking at the options to change the settings and I can change the application settings from the Web Panel directly.
Is there a way I can change the server settings from the panel as well? If not, then how should I change them?

Also, are they the same for Standalone and Cluster mode or is it different?

Thank you.

**Comment by yashtandon113** [comment]
@Mohit-3196 
Thanks for your question.

For the time being, server settings cannot be altered using the web panel, but I believe we will be able to do so in the future. There are now two methods for changing the server settings.

**1. Change via SSH into the server:**
In order to edit the server settings like change port etc, edit the `red5.properties` filer under `/usr/local/antmedia` folder.

Their are multiple settings for server configuration, please check out [here](https://antmedia.io/docs/guides/configuration-and-testing/ams-server-configuration/).

After making the change please restart the server.

**2. Change using Management Rest API:**
Here is the Rest API to change the server settings: https://antmedia.io/rest/#/ManagementRestService/changeServerSettings

Here is the postman curl sample for the same. In this example, I made the `GlobalIp` to true so you need to only mention the setting that you would like to change.

```
curl --location 'http://localhost:5080/rest/v2/server-settings' \
--header 'Content-Type: application/json' \
--header 'Cookie: JSESSIONID=343931B6A55B18C42F2A1D083743C481' \
--data '{
    "useGlobalIp": true
}'
```

In order to call management Rest APIs, you need to authenticate first. Please check the [following document](https://antmedia.io/docs/guides/developer-sdk-and-api/rest-api-guide/management-rest-apis/).

The way is same for both standalone and cluster mode.
